### ISSUE TITLE
"Target class [mail.manager] does not exist" error

### ISSUE THREAD

**Issue by wonder95** [wouterdeberg/laravel-elastic-email#9]
I am attempting to use this with a Laravel Breeze (v9) install, and when I follow the instructions and attempt to use the password reset2 functionality, I get the error
```
Target class [mail.manager] does not exist.
```
The erroring code is at this line
```
        $status = Password::sendResetLink(
            $request->only('email')
        );
```
which is using the `Password` facade. The password reset just sends a notification, and it doesn't appear to have the built in `MailServiceProvider` specified.

I've verified that I'l followed the steps in the README. Is there another step I could be missing?

**Comment by wonder95** [comment]
The issue was that I followed the instructions as they are that say to `comment out Laravel's default MailServiceProvider`. Apparently those are v1 instructions, as it was pointed out to me that below that, it says `Turn the MailServiceProvider back on in your app.php config file` in the `Upgrade Guide from V1 to V2` section. Once I did that, it worked.

I would suggest cleaning up those instructions somehow.
